auth/api/iam/openid4vp.go

```diff
@@ -542,10 +545,21 @@ func (r Wrapper) handleAccessTokenRequest(ctx context.Context, verifier did.DID,
 
 	presentations := oauthSession.ServerState.VerifiablePresentations()
 	submission := oauthSession.ServerState.PresentationSubmission()
-	definition, err := r.policyBackend.PresentationDefinition(ctx, verifier, oauthSession.Scope)
+	definitions, err := r.policyBackend.PresentationDefinitions(ctx, verifier, oauthSession.Scope)
 	if err != nil {
 		return nil, withCallbackURI(oauthError(oauth.ServerError, fmt.Sprintf("failed to fetch presentation definition: %s", err.Error())), callbackURI)
 	}
+	// todo, for now take the organization definition
+	var definition *pe.PresentationDefinition
+	for _, def := range definitions {
+		if def.AudienceType == pe.AudienceTypeOrganization {
```

Comment:
third time, maybe add a helper function to the policy interface, or an optional audience parameter to `PresentationDefinitions()`?

Reply:
this should be temporary until there's an actual user wallet to call.